config/utilities/uenv.py

```diff
@@ -100,13 +109,9 @@ def _get_uenvs():
 
             if isinstance(activation, list):
                 env['prepare_cmds'] = activation
-            # no longer support implicit activation scripts
+            # FIXME: this is deprecated and should be removed
```

Comment:
Is it possible to print a loud warning?
Or it could a "feature" - if the user provides an argument of this form, it should be sourced?
If it's a feature, you could check that `activation` is a valid path name?

Reply:
It is possible to print a warning but since the warnings are handled by the ReFrame framework itself this will look out of place. I would say that we leave it like this so we can still test old images and once we upgrade all of them we remove it. No need to have a feature.